workers/data-url-shared.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<title>data URL shared worker</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id=log></div>
+<script>
+var worker = new SharedWorker("data:,onconnect = (e) => { fetch('/').then(() => e.ports[0].postMessage('fail'), () => e.ports[0].postMessage('pass')) }") // not same-origin
```

Comment:
https://fetch.spec.whatwg.org/ says that HTML "prevents creation of shared workers from such URLs." but that doesn't seem to match https://html.spec.whatwg.org/#dom-sharedworker

If they shouldn't be created at all, then presumably the constructor should throw?